config/mapping.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
I'd rather see us extracting that JSON blob from `ember-modules-codemod` into a separate module, e.g. `ember-rfc176-data`